regexes/client/mobile_apps.yml

```diff
@@ -54,17 +54,17 @@
   version: '$1'
 
 # Facebook Messenger (https://play.google.com/store/apps/details?id=com.facebook.orca)
-- regex: '(?:MessengerForiOS|MESSENGER|FB_IAB/Orca-Android).(?:FBAV)(?:[ /]([\d\.]+))?'
+- regex: '(?:MessengerForiOS|MESSENGER|FB_IAB/Orca-Android)\.(?:FBAV)(?:[ /]([\d\.]+))?'
   name: 'Facebook Messenger'
   version: '$1'
 
 # Facebook Messenger Lite
-- regex: '(?:mLite|MessengerLite(?:ForiOS)?).(?:FBAV)(?:[ /]([\d\.]+))?'
+- regex: '(?:mLite|MessengerLite(?:ForiOS)?)\.(?:FBAV)(?:[ /]([\d\.]+))?'
   name: 'Facebook Messenger Lite'
   version: '$1'
 
 # Facebook Groups
-- regex: '(?:GroupsForiOS).(?:FBAV)(?:[ /]([\d\.]+))?'
+- regex: '(?:GroupsForiOS)\.(?:FBAV)(?:[ /]([\d\.]+))?'
```

Comment:
```suggestion
- regex: '(?:GroupsForiOS).*FBAV(?:[ /]([\d\.]+))?'
```

Reply:
> Hm looking at the action logs, I don't actually see why the `*` is required here, all the cases I find have `GroupsForiOS;FBAV`. Can you point me at a failing test that needs to match `.*` here please?


There might be some cases like this:

`Mozilla/5.0 (Linux; Android 11; SM-G973F Build/RP1A.200720.012; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/95.0.4638.74 Mobile Safari/537.36 [FBAN/AudienceNetworkForAndroid;FBSN/Android;FBSV/11;FBAB/com.dopuz.klotski.riddle;FBAV/5.1701;FBBV/51701;FBVS/6.7.0;FBLC/en_GB]` where `FBAV` isn't right after app name.

Reply:
Okay! Thanks

Reply:
1de135493ef5767cd9433bc50afb4cfe1c2b6d2b